homeassistant/components/cast/media_player.py

```diff
@@ -87,21 +87,8 @@
 
 
 ENTITY_SCHEMA = vol.All(
-    cv.deprecated(CONF_HOST, invalidation_version="0.116"),
     vol.Schema(
         {
-            vol.Exclusive(CONF_HOST, "device_identifier"): cv.string,
-            vol.Exclusive(CONF_UUID, "device_identifier"): cv.string,
-            vol.Optional(CONF_IGNORE_CEC): vol.All(cv.ensure_list, [cv.string]),
-        }
-    ),
-)
-
-PLATFORM_SCHEMA = vol.All(
-    cv.deprecated(CONF_HOST, invalidation_version="0.116"),
-    PLATFORM_SCHEMA.extend(
-        {
-            vol.Exclusive(CONF_HOST, "device_identifier"): cv.string,
             vol.Exclusive(CONF_UUID, "device_identifier"): cv.string,
```

Comment:
This can be removed too.

Reply:
That one hasn't been deprecated? I think the diff looks weird and might have thrown you off?

The UUID for the entity schema remains, the platform schema is completely removed.

Reply:
Something happened, my comment was not intended for that row.